Clarification of CNF Conformance Program, adding CNF Working Group info (#450)
cncf/cnf-conformance: cnf-wg/README.md

# Cloud Native Network Function Working Group (CNF-WG)

> The CNF WG operates under the aegis of CNCF. The charter of the working group is to define the process around certifying the cloud nativeness of telco applications,  aka CNFs. We collaborate with the [CNF Conformance test sutie project](README-testsutie.md) who work on the mechanics of the conformance tests.

Reviewer: ```suggestion
The CNF WG operates under the aegis of CNCF. The charter of the working group is to define the process around certifying the cloud nativeness of telco applications,  aka CNFs. We collaborate with the [CNF Conformance test suite project](README-testsuite.md) who work on the mechanics of the conformance tests.
```

> The goal for the group is to create a software conformance program that any telco application implementation can use to demonstrate that they are conformant and interoperable with cloud native principles.

Reviewer 2: telco application --> network application (telco is a sub-variant in this domain. we call out in the charter that large scale enterprises *who maintain their own large networks* will be a target audience as well).  We are calling them CNFs as opposed to CTF.

Author: I like that terminology. I think there will need to be adjustments in the rest of the doc to talk about telco as a subset of the network domain.

> The goal for the group is to create a software conformance program that any telco application implementation can use to demonstrate that they are conformant and interoperable with cloud native principles.

Author: ```suggestion
The goal for the group is to create a software conformance program that any network application implementation can use to demonstrate that it is conformant and interoperable with cloud native principles.
```
